main.js

```diff
@@ -1 +1,55 @@
 import customers from './customers.js'
+
+let person = [{
+  thumbNail:
+    'https://randomuser.me/api/portraits/med/men/88.jpg',
+  name: "Jose Rhodes",
+  email: "jose.rhodes@example.com",
+  address: "3347 hickory creek Dr, Helena, AR, 81276",
+  'dob': '1955-11-10',
+}]
+
+function listCustomers(personArray) {
+  let customerList = document.querySelector('#customer-list')
+    ;
+  console.log(customerList)
+
+
+  for (let customer of customers) {
```

Comment:
Here you'll want to loop over `personArray`, which is the name that the array has inside the function. You can specify which array when you call the function.

`listCustomers(customers)`